task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml

```diff
@@ -16,43 +16,45 @@ spec:
     When [Java dependency rebuild](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/cli/proc_enabled_java_dependencies.html) is enabled it triggers rebuilds of Java artifacts.
```

Comment:
This code is now generated from two different places, as the remote task generation is expecting to generate this from the buildah-oci-ta.

Reply:
From what I understand the `hack/generate-buildah-remote.sh` script will generate `task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml`, there is no `recipe.yaml` in `task/buildah-remote-oci-ta/0.1` so tash and the `hack/generate-ta-tasks.sh` script will not try to generate this file.

Reply:
ok, I missed that it skipped this one, looks good to me.

I have a PR open at https://github.com/konflux-ci/build-definitions/pull/1068 that moves this generator to the build-definitions repo.

Reply:
Yeah I saw that, I think we'll move this from ec/hacks here as well.